backend/src/graphql/resolver/TransactionResolver.ts

```diff
@@ -125,6 +127,7 @@ export const executeTransaction = async (
   }
 
   // send notification email
+  const overviewURL = new URL('/overview', CONFIG.EMAIL_CLIENT_URI)
```

Comment:
please specify the full URL in the config parameter.
Consider to name the config value properly - its not an email client, but the `EMAIL_WALLET_URI`

Reply:
I like to discuss this with you personally @ulfgebhardt .

Reply:
Ulf explaind me his reasons and I followed his suggestions.